oci.go

```diff
@@ -20,14 +20,15 @@ import (
 
 // OCI config file
 const (
-	ociConfigFile     string      = "config.json"
-	ociConfigFileMode os.FileMode = 0444
+	ociConfigFile       string      = "config.json"
+	ociConfigFileMode   os.FileMode = 0444
+	ociConfigBundlePath string      = "/run/kata-containers/sandbox"
```

Comment:
mmm not sure if this is a good idea, config.json is container specific, so in the case of a POD with 2 or more containers, do they share the same config.json ?

Reply:
@devimc, yeah good catch. They won't share the config.json. I think putting it under sandbox id will be needed.

Reply:
@devimc updated the patch.. please take a look and let me know if you see any issues